### PR TITLE
Update UG to increase clarity on how to cd

### LINKED ISSUE #348: [PE-D][Tester B] No glossary of terms in the user guide
There is no glossary of terms in the user guide. This will make it tough for non-technical users to quickly understand how to use the app.

For instance, some may not known the meaning of CLI or "cd into the folder you put the jar file in".

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: Runus01/ped#7

### LINKED ISSUE #356: [PE-D][Tester B] Missing full stop in user guide and developer guide
There is a missing full stop under the "Deleting a group" and "Searching persons by field" section of the user guide. Also, in the "Non-Functional Requirements" of the developer guide.

**User Guide**

**Developer Guide**

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: Runus01/ped#9

### PR DESCRIPTION
**Pull request by seah-minlong**
* Fixes #348 
* Fixes #356

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -27,12 +27,24 @@
...
+4. Open a command terminal, `cd` into the folder you put the jar file in.
+
+<box type="tip" seamless>
+
+* `cd` is the command to change directories.
+* e.g. `cd Documents/contactsforgood`
+
+</box> 
+
+5. Use the `java -jar contactsforgood.jar` command to run the application.
```
Comment (voldiemort) [inline docs/UserGuide.md:39]: I have noted that we do have
`1. Ensure you have Java 17 or above installed in your Computer.`
in the guide, but should we also tell the user to use `java -version` to check that they are running Java 17, much like how we conduct our smoke tests? We can tell them to refer back to `1.` if the version shown is lower than 17.

Or would that be too technical for an average user? Lmk your thoughts.

**Review by voldiemort — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (seah-minlong) [reply docs/UserGuide.md:39]: Sounds good

File: docs/UserGuide.md [hunk 2]
```diff
@@ -21,18 +21,30 @@
...
 ## Quick start
 
-1. Ensure you have Java `17` or above installed in your Computer.
+1. Download the latest `.jar` file from [here](https://github.com/AY2425S1-CS2103T-T16-3/tp/releases).
 
-2. Download the latest `.jar` file from [here](https://github.com/AY2425S1-CS2103T-T16-3/tp/releases).
+2. Copy the file to the folder you want to use as the _home folder_ for CFG.
 
-3. Copy the file to the folder you want to use as the _home folder_ for CFG.
+3. Open a command terminal. Run the `java -version` command to ensure you are using Java 17.
```
[the remark this answers is not in the document]
Comment (seah-minlong) [reply docs/UserGuide.md:28]: image
Is it supposed to be strictly 17?

[the remark this answers is not in the document]
Comment (voldiemort) [reply docs/UserGuide.md:28]: Hmm, the original Step 1. in our UG said Java 17 or above, but the CS2103T website does say Java 17. Let's stick to Java 17 then.

**Pull request merged**